Source/Other/Extensions/UserDefaultsExtension.swift

```diff
@@ -29,7 +29,8 @@ extension UserDefaults {
 		return savedFont
 	}
 
-    @objc dynamic var SPSecureBookmarks: [Dictionary<String, Data>] {
+    // needs to be objc for KVO
+    @objc var SPSecureBookmarks: [Dictionary<String, Data>] {
         return array(forKey: SASecureBookmarks) as? [Dictionary<String, Data>] ?? [["noBookmarks": Data()]]
```

Comment:
can we return empty array as fallback then? empty array will have check for isNotEmpty

Reply:
`return array(forKey: SASecureBookmarks) as? [Dictionary<String, Data>] ?? []`